components/detailMenus/TransformProperty.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+    <div
+        class="text-center"
+        style="width:100%;"> 
+        <div class="uploadBTN py-3" style="align-center">
+            <v-row
+                align="center"
+                justify="center">
```

Comment:
Please check coding style. And please follow it.
It seems that other places are using following coding style.
            <v-row
                align="center"
                justify="center"
            >